### ISSUE TITLE
topics_over_time only works for nr_topics = None

### ISSUE THREAD

**Issue by sgdantas** [MaartenGr/BERTopic#96]
Hey, first of all, great work!

I noticed the `BERTopic.topics_over_time` method only works when the argument `nr_topics` is set to None.

We can use the trump dataset to reproduce it

```
import re
import pandas as pd
from bertopic import BERTopic
trump = pd.read_csv('https://drive.google.com/uc?export=download&id=1xRKHaP-QwACMydlDnyFPEaFdtskJuBa6')
trump.text = trump.apply(lambda row: re.sub(r"http\S+", "", row.text).lower(), 1)
trump.text = trump.apply(lambda row: " ".join(filter(lambda x:x[0]!="@", row.text.split())), 1)
trump.text = trump.apply(lambda row: " ".join(re.sub("[^a-zA-Z]+", " ", row.text).split()), 1)
trump = trump.loc[(trump.isRetweet == "f") & (trump.text != ""), :]
timestamps = trump.date.to_list()
tweets = trump.text.to_list()
model = BERTopic(verbose=True)
topics, _ = model.fit_transform(tweets)
model = BERTopic(nr_topics='auto',verbose=True)
topics, _ = model.fit_transform(tweets)
topics_over_time = model.topics_over_time(tweets, topics, timestamps, nr_bins=30)
```

![image](https://user-images.githubusercontent.com/34652852/115032521-6532a400-9e97-11eb-8050-1345f256b651.png)

The number of topics is decreased but the indexing is not updated, it should be easy to fix it

Best

**Comment by MaartenGr** [comment]
Thank you for opening this issue. I will look into it and make it is fixed in the next major release!

**Comment by MaartenGr** [comment]
@sgdantas This is now fixed in the new release of BERTopic (v0.7 #87). Update bertopic with `pip install --upgrade bertopic` and you should be good to go!

**Comment by sgdantas** [comment]
Awesome, thanks a lot @MaartenGr and congrats once again for your work!
